src/calculateDeepEqualDiffs.js

```diff
@@ -127,16 +127,53 @@ function accumulateDeepEqualDiffs(a, b, diffsAccumulator, pathString = '', { det
     return trackDiff(a, b, diffsAccumulator, pathString, diffTypes.function);
   }
 
-  if (isPlainObject(a) && isPlainObject(b)) {
-    const keys = getKeys(a);
+  if (isError(a) && isError(b)) {
+    const keys = Object.getOwnPropertyNames(a);
+    if (keys.length !== Object.getOwnPropertyNames(b).length) {
+      return trackDiff(a, b, diffsAccumulator, pathString, diffTypes.different);
+    }
+    // Do not compare the stack as it might differ even though the errors are identical.
+    const relevantKeys = keys.filter(k => k !== 'stack');
```

Comment:
let's move the `relevantKeys` calculation in to the
```
if (typeof a === 'object' && typeof b === 'object' && Object.getPrototypeOf(a) === Object.getPrototypeOf(b)) {
```
scope

so we won't repeat the rest of the code.